apps/front/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "front",
+  "type": "module",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx ",
```

acá no te faltan todos los flags que ya estaban antes ?
```suggestion
    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
```